src/lib/storage/index/group_key/variable_length_key_base.cpp

```diff
@@ -69,32 +69,34 @@ VariableLengthKeyBase& VariableLengthKeyBase::shift_and_set(uint64_t value, uint
   return *this;
 }
 
-bool operator==(const VariableLengthKeyBase& lhs, const VariableLengthKeyBase& rhs) {
```

Comment:
`lhs` and `rhs` are fairly common abbreviatinos. Why change it?

Reply:
I'm fine with lhs / rhs as well, but it should be consistent everywhere. I just realized that I missed some places. Unless you're a big fan of lhs, I would change the rest as well.